<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qlbt</class>
 <widget class="QMainWindow" name="qlbt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QUẢN LÝ BÁN TRUYỆN</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>0</y>
      <width>321</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; color:#ffffff;&quot;&gt;QUẢN LÝ BÁN TRUYỆN&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_timkiembt">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TÌM KIẾM</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditBanTruyen">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>411</width>
      <height>192</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>MÃ ĐB</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>TÊN KH</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>NGÀY BÁN</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>TRẠNG THÁI</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btn_taobt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TẠO ĐƠN BÁN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ctbt">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>320</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>XEM CHI TIẾT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_thanhtoan">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>320</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>THANH TOÁN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_xoabt">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>320</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>XÓA ĐƠN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_quaylai4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>QUAY LẠI</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_suabt">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>320</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>SỬA ĐƠN</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-50</y>
      <width>1051</width>
      <height>881</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../anhnen/R.jpg</pixmap>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>label</zorder>
   <zorder>btn_timkiembt</zorder>
   <zorder>lineEditBanTruyen</zorder>
   <zorder>tableWidget</zorder>
   <zorder>btn_taobt</zorder>
   <zorder>btn_ctbt</zorder>
   <zorder>btn_thanhtoan</zorder>
   <zorder>btn_xoabt</zorder>
   <zorder>btn_quaylai4</zorder>
   <zorder>btn_suabt</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>621</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
